jujugui/static/gui/src/app/app.js

```diff
@@ -2927,6 +2901,7 @@ YUI.add('juju-gui', function(Y) {
         console.log('switching models is not supported in sandbox');
       }
       console.log('switching model connection');
+      this.env.loading = true;
```

Comment:
So when we call switch env we set this  property to `true` to mark it as loading. But the only way we can get it to be in a not loading state is if root is `new`? Or is there somewhere else that modifies this property that I don't see?